config.toml

```diff
@@ -292,6 +292,10 @@ logo = "mivation-inc.png"
 url = "https://www.purefacts.com"
 logo = "purefacts.png"
 
+[[params.users]]
```

Comment:
Can you move it up the list so that it's alphabetically ordered please?

Reply:
Thanks @tomkerkhove - fixed